universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRenderer.java

```diff
@@ -85,4 +82,21 @@ Tag renderInternal(
         .with(questionTextDiv)
         .with(questionFormContent);
   }
+
+  private ContainerTag createQuestionTextTag(Messages messages) {
```

Comment:
I don't see any new tests for showing the correct thing in the correct instance.